src/Sdk/DTWebApi/WebApi/TimelineRecordLogLine.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Runtime.Serialization;
+
+namespace GitHub.DistributedTask.WebApi
+{
+    [DataContract]
+    public sealed class TimelineRecordLogLine
+    {
+        public TimelineRecordLogLine(String line, long lineNumber)
+        {
+            this.Line = line;
+            this.LineNumber = lineNumber;
+        }
+
+        [DataMember]
+        public String Line
+        {
+            get;
+            set;
+        }
+
+        [DataMember]
+        public long LineNumber
+        {
+            get;
+            set;
+        }
+
+        public override bool Equals(object obj)
+        {
+            TimelineRecordLogLine logLine = obj as TimelineRecordLogLine;
+            
+            if(logLine != null)
+            {
+                return String.Equals(Line, logLine.Line) 
+                       && LineNumber.Equals(logLine.LineNumber);
+            }
+
+            return false;
+        }
+
+        public override int GetHashCode()
```

Comment:
do we add to a dictionary? if not, do we need this?

if needed hash code should be immutable, so might need to change Line/LineNumber setter to private

will look more later, but on call this week and ting is out. Might be good to pair with someone on runtime so they have context since you'll be out.

Reply:
Lokesh may have context in this area already

Reply:
It's not used. We can git rid of this too.